crates/matrix-sdk/README.md

```diff
@@ -1,11 +1,28 @@
-A [Matrix](https://matrix.org/) client library written in Rust.
+A high-level, batteries-included [Matrix](https://matrix.org/) client library
+written in Rust.
 
-The matrix-sdk aims to be a general purpose client library for writing Matrix
-clients, bots, and other Matrix related things that use the client-server API to
-communicate with a Matrix homeserver.
+This crate seeks to be a general-purpose library for writing software using the
+Matrix [Client-Server API](https://matrix.org/docs/spec/client_server/latest)
+to communicate with a Matrix homeserver. If you're writing a typical Matrix
+client or bot, this is likely the crate you need.
 
-# Examples
-Connecting and logging in to a homeserver is pretty starightforward:
+However, the crate is designed in a modular way and depends on several
```

Comment:
I don't think this should be part of the README/crate docs. The majority of users are here to make bots or clients, yet they get presented with implementation details of the crate right away.

I think the README of the repo, where we have such a list, is a better place for this.